[2023-10-17 07:57:42,887] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: currency_btc_rub_every_10_munutes.get_rate_from_api_task manual__2023-10-17T07:52:38.326627+00:00 [queued]>
[2023-10-17 07:57:42,907] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: currency_btc_rub_every_10_munutes.get_rate_from_api_task manual__2023-10-17T07:52:38.326627+00:00 [queued]>
[2023-10-17 07:57:42,908] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-17 07:57:42,909] {taskinstance.py:1377} INFO - Starting attempt 2 of 2
[2023-10-17 07:57:42,910] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-17 07:57:42,932] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_rate_from_api_task> on 2023-10-17 07:52:38.326627+00:00
[2023-10-17 07:57:42,943] {standard_task_runner.py:52} INFO - Started process 941 to run task
[2023-10-17 07:57:42,949] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'currency_btc_rub_every_10_munutes', 'get_rate_from_api_task', 'manual__2023-10-17T07:52:38.326627+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/hello.py', '--cfg-path', '/tmp/tmpo_piq8p3', '--error-file', '/tmp/tmppexf6ccq']
[2023-10-17 07:57:42,952] {standard_task_runner.py:80} INFO - Job 47: Subtask get_rate_from_api_task
[2023-10-17 07:57:43,049] {task_command.py:371} INFO - Running <TaskInstance: currency_btc_rub_every_10_munutes.get_rate_from_api_task manual__2023-10-17T07:52:38.326627+00:00 [running]> on host 5ae209acedc4
[2023-10-17 07:57:43,154] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Andrey1309
AIRFLOW_CTX_DAG_ID=currency_btc_rub_every_10_munutes
AIRFLOW_CTX_TASK_ID=get_rate_from_api_task
AIRFLOW_CTX_EXECUTION_DATE=2023-10-17T07:52:38.326627+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-17T07:52:38.326627+00:00
[2023-10-17 07:57:43,157] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: get_rate_from_api() takes 0 positional arguments but 6 were given
[2023-10-17 07:57:43,170] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=currency_btc_rub_every_10_munutes, task_id=get_rate_from_api_task, execution_date=20231017T075238, start_date=20231017T075742, end_date=20231017T075743
[2023-10-17 07:57:43,203] {standard_task_runner.py:97} ERROR - Failed to execute job 47 for task get_rate_from_api_task (get_rate_from_api() takes 0 positional arguments but 6 were given; 941)
[2023-10-17 07:57:43,247] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-17 07:57:43,313] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
